src/model/yaml/docker-compose/caComposeYaml.ts

```diff
@@ -84,7 +83,7 @@ class CaDockerComposeYaml extends DockerComposeYaml {
       volumes: [
         `\${BDK_DOCKER_HOST_PATH:-~/.bdk}/${config.networkName}/ca:/fabric-ca`,
       ],
-      user: `${os.userInfo().uid}:${os.userInfo().gid}`
+      user: `${config.UID}:${config.GID}`
```

Comment:
mac上產出的docker-compose是
```
user: 502:20
```
docker-compose up時會出現
```
ERROR: The Compose file '/Users/eugene/.bdk/bdk-network/docker-compose/docker-compose-ca-rca.bdk.cathaybc.com.yaml' is invalid because:
services.rca.bdk.cathaybc.com.user contains an invalid type, it should be a string
```

Reply:
fe80ba2

Reply:
solve it in #8